topics/admin/tutorials/ansible-galaxy/group_vars/galaxyservers.yml

```diff
@@ -38,6 +38,26 @@ galaxy_config:
     check_migrate_tools: false
     tool_data_path: "{{ galaxy_mutable_data_dir }}/tool-data"
     job_config_file: "{{ galaxy_config_dir }}/job_conf.xml"
+    # SQL Performance
+    database_engine_option_server_side_cursors: true
+    slow_query_log_threshold: 5
+    enable_per_request_sql_debugging: true
+    # File serving Performance
+    nginx_x_accel_redirect_base: /_x_accel_redirect
+    # Automation / Ease of Use
+    watch_job_rules: 'auto'
+    allow_path_paste: true
+    enable_quotas: true
+    allow_user_deletion: true
+    show_welcome_with_login: true
+    expose_user_name: true
```

Comment:
I wouldn't add this, since it probably violates the GDPR (not that I'm advocating to add `enable_beta_gdpr: true` here).

Reply:
Yeah. I really wish we had a lawyer to comment on that but in retrospect it is a "Public User Name" (per the registration form), there should be a strong expectation that it's public.  I was too conservative in the original implementation and I regret that.